src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -142,6 +142,16 @@ public VerboseProcess(final Process prc, final Level stdout,
         if (stderr == null) {
             throw new IllegalArgumentException("stderr LEVEL can't be NULL");
         }
+        if (Level.ALL.equals(stdout)) {
+            throw new IllegalArgumentException(
+                "stdout LEVEL can't be set to ALL"
+            );
+        }
+        if (Level.ALL.equals(stderr)) {
+            throw new IllegalArgumentException(
+                "stderr LEVEL can't be set to ALL"
```

Comment:
@mkordas do you think it is really necessary? Above this code you have:

        if (prc == null) {
            throw new IllegalArgumentException("process can't be NULL");
        }
        if (stdout == null) {
            throw new IllegalArgumentException("stdout LEVEL can't be NULL");
        }
        if (stderr == null) {
            throw new IllegalArgumentException("stderr LEVEL can't be NULL");
        }

and the `can't be NULL` is duplicated as well. I think extracting common part only darkened image. But if you really want... :)